packages/lambda-powertools-firehose-client/index.js

```diff
@@ -1,18 +1,5 @@
+process.env.AWS_NODEJS_CONNECTION_REUSE_ENABLED = '1'
 const AWS = require('aws-sdk')
```

Comment:
I wonder if we could require the Firehose client rather than the whole SDK? WDYT?
(Same goes for the Kinesis service)

Reply:
Good shout, we should do that with all the client libs, help reduce cold start duration